diff --git a/toolchain/cc_toolchain_config.bzl b/toolchain/cc_toolchain_config.bzl
index 1f0395b..c468866 100644
--- a/toolchain/cc_toolchain_config.bzl
+++ b/toolchain/cc_toolchain_config.bzl
@@ -12,10 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.

+# NOTE(storypku): generated_symlink local_config_cc//:cc_toolchain_config.bzl
 load(
-    "@bazel_tools//tools/cpp:unix_cc_toolchain_config.bzl",
+    "@rules_cc//cc/private/toolchain:unix_cc_toolchain_config.bzl",
     unix_cc_toolchain_config = "cc_toolchain_config",
 )
+
 load(
     "//toolchain/internal:common.bzl",
     _check_os_arch_keys = "check_os_arch_keys",
@@ -275,7 +277,7 @@ def cc_toolchain_config(
     # `lld` is being used as the linker.
     supports_start_end_lib = use_lld

-    # Source: https://cs.opensource.google/bazel/bazel/+/master:tools/cpp/unix_cc_toolchain_config.bzl
+    # Source: https://github.com/bazelbuild/rules_cc/blob/main/cc/private/toolchain/unix_cc_toolchain_config.bzl
     unix_cc_toolchain_config(
         name = name,
         cpu = target_cpu,
@@ -299,5 +301,4 @@ def cc_toolchain_config(
         coverage_compile_flags = coverage_compile_flags,
         coverage_link_flags = coverage_link_flags,
         supports_start_end_lib = supports_start_end_lib,
-        builtin_sysroot = sysroot_path,
     )
